.github/workflows/ci.yml

```diff
@@ -70,11 +70,11 @@ jobs:
       - name: Run stubs check
         run: python3 tests/stubs_check.py
 
-      - name: Run examples check
-        run: python3 tests/examples_check.py
-
       - name: Install pytest
-        run: pip3 install pytest pytest-xdist
+        run: pip3 install pytest pytest-xdist fixtures
+
+      - name: Run examples check
```

Comment:
Why do you need another step for examples check? Isn't it also executed as part of next `pytest` command?

Reply:
It looks like the pytest step doesn't auto detect the tests in examples

Reply:
I could change the name of `tests/examples_check.py` to `tests/test_examples.py` and it would be picked up by the pytest step. Should the same be done for `stubs_check.py` and have a single pytest call that calls all of them? Maybe having them in their own steps might be helpful for debugging. What do you think?